cost/azure/savings_plan/utilization/azure_savings_plan_utilization.pt

```diff
@@ -0,0 +1,520 @@
+name "Azure Savings Plan Utilization"
+rs_pt_ver 20180301
+type "policy"
+short_description "Reports the utilization of Azure Savings Plans within the Azure estate. NOTE: These Utilization metrics are generated by Azure. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/azure/savings_plan/utilization) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "medium"
+category "Cost"
+default_frequency "monthly"
+info(
+  version: "0.1.0",
+  provider: "Azure",
+  service: "Compute",
+  policy_set: "Savings Plans"
+)
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  category "Policy Settings"
+  label "Email Addresses"
+  description "Email addresses of the recipients you wish to notify"
+  default []
+end
+
+parameter "param_azure_endpoint" do
+  type "string"
+  category "Policy Settings"
+  label "Azure Endpoint"
+  description "Select the API endpoint to use for Azure. Use default value of management.azure.com unless using Azure China."
+  allowed_values "management.azure.com", "management.chinacloudapi.cn"
+  default "management.azure.com"
+end
+
+parameter "param_utilization_threshold" do
+  type "number"
+  category "Policy Settings"
+  label "Maximum Savings Plan Utilization Threshold"
```

Comment:
(%) should be in the label, not the description, to match common practice in other policies.

(README should be updated accordingly as well)